rtslib/node.py

```diff
@@ -52,9 +52,16 @@ def _create_in_cfs_ine(self, mode):
         '''
         if mode not in ['any', 'lookup', 'create']:
             raise RTSLibError("Invalid mode: %s" % mode)
+
         if self.exists and mode == 'create':
-            raise RTSLibError("This %s already exists in configFS"
-                              % self.__class__.__name__)
+            # ensure that self.path is not stale hba-only dir
+            if os.path.samefile(os.path.dirname(self.path), self.configfs_dir+'/core') \
```

Comment:
oops, actually I have a quibble. self.path is likely going to be something like `/sys/kernel/config/target/core/user_33` so I don't think samefile() of this against `/sys/kernel/config/target/core` will return true when we want it to.

Reply:
thanks for the explanation, I am always learning things 😃 Let me go ahead and merge this, then.

Would a new release tag be helpful to you..?

Reply:
@agrover Glad that I was able to explain it right :-)

> Would a new release tag be helpful to you..?

I will wait for @maurizio-lombardi to respond on this.

Thanks for all the help!

Reply:
Yes, a new release tag would be helpful. Thanks!